<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>742</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="leftPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>720</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="eraserButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pencilButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightPanel">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>150</width>
      <height>720</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="secondaryBrushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="primaryBrushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBrushesButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="swapBrushesButton">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="findAndReplaceButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bucketButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorPicker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="framesScroll">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>340</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="framesScrollWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>139</width>
       <height>259</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="background_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="background_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrameButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="playbackFrame">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>380</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="actualSizeButton">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="animationLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>66</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>201</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>per second</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>183</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Frames</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="fpsBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>193</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="duplicateFrameButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>610</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="rightButtonsFrame">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="redoButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="undoButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="newProjectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>leftPanel</zorder>
   <zorder>eraserButton</zorder>
   <zorder>pencilButton</zorder>
   <zorder>rightPanel</zorder>
   <zorder>resetBrushesButton</zorder>
   <zorder>swapBrushesButton</zorder>
   <zorder>findAndReplaceButton</zorder>
   <zorder>bucketButton</zorder>
   <zorder>colorPicker</zorder>
   <zorder>addFrameButton</zorder>
   <zorder>framesScroll</zorder>
   <zorder>background_3</zorder>
   <zorder>secondaryBrushButton</zorder>
   <zorder>background_2</zorder>
   <zorder>primaryBrushButton</zorder>
   <zorder>playbackFrame</zorder>
   <zorder>deleteFrameButton</zorder>
   <zorder>duplicateFrameButton</zorder>
   <zorder>rightButtonsFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
